requirements.txt

```diff
@@ -0,0 +1,7 @@
+banal==1.0.6
```

Comment:
I know I'm backtracking on this, but since `servicelayer` is a library perhaps we can do less strict pinning here. For instance 

```
banal >=1.0.6, <2.0.0
```

Reply:
Actually now that I see it perhaps what we had in `setup.py` was very close to that, so I would go with:

```
banal >= 1.0.6, < 2.0.0
normality >= 2.4.0, < 3.0.0
fakeredis == 1.10.1, < 2.0.0
sqlalchemy >= 2.0.4, < 3.0.0
structlog >= 20.2.0, < 23.0.0
colorama >= 0.4.4, < 1.0.0
pika >= 1.3.1, < 2.0.0
```